packages/wallet/src/SingleAddressWallet.ts

```diff
@@ -337,6 +337,8 @@ export class SingleAddressWallet implements ObservableWallet {
     this.txSubmitProvider.stats.shutdown();
     this.networkInfoProvider.stats.shutdown();
     this.stakePoolProvider.stats.shutdown();
+    this.utxoProvider.stats.shutdown();
```

Comment:
The inclusion of `this.utxoProvider.stats.shutdown();` is a separate fix commit we'll merge to master. The other changes in this PR can be squashed into the original commit.

Reply:
#268 has this fix too, so you could just remove it from this PR